src/LiteMember.md

```diff
@@ -26,7 +26,17 @@ Config config = new Config();
 config.setLiteMember(true);
 ```
 
-![image](images/NoteSmall.jpg) ***NOTE:*** *Note that you cannot change a member's role at runtime.*
+#### Promoting Lite Members to Data Member
 
+Lite members can be promoted to data members using `Cluster` interface. When they are promoted, cluster partitions are rebalanced and ownerships of some portion of the partitions are assigned to newly promoted data members. A data member cannot be downgraded to a lite member back.
 
```

Comment:
@mdogan, the last sentence is repeated as a note at the bottom of the section. Is it just to emphasize?